### PR TITLE
Change all occurrence of election into form

### PR DESCRIPTION
**Pull request by Ghita2002**
I change election to form. I need to do other changes like changing the image in the home page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,10 +256,10 @@ The shuffling service ensures that encrypted votes can not be linked to the user
 who cast them. Once the service is setup, each node can perform what we call a
 "shuffling step". A shuffling step re-orders an array of elements such that the
 integrity of the elements is guarantee (i.e no elements have been modified,
-added, or removed), but one can't trace how elements have been re-ordered. 
+added, or removed), but one can't trace how elements have been re-ordered.
 
 In D-Voting we use the Neff [[2]] implementation of verifiable shuffling. Once
-an election is closed, an admin can trigger the shuffling steps from the nodes.
+an form is closed, an admin can trigger the shuffling steps from the nodes.
```
Comment (nkcr) [inline README.md:262]:
```suggestion
a form is closed, an admin can trigger the shuffling steps from the nodes.
```

File: README.md [hunk 2]
```diff
@@ -107,11 +107,11 @@ be linked to the original voter.
     <img height="50px" src="docs/assets/audit-white.png#gh-dark-mode-only">
 </div>
 
-**Transparency/Verifiability/Auditability** - The whole election process is
+**Transparency/Verifiability/Auditability** - The whole form process is
```
Comment (nkcr) [inline README.md:110]: a "form process" doesn't make much sense, compared to the previous "election process". We should come with a better naming. In this case we might say the "voting process". 
(same in other places)

File: README.md [hunk 3]
```diff
@@ -120,12 +120,12 @@ The project has 4 main high-level components:
 **Proxy** - A proxy offers the mean for an external actor such as a website to
 interact with a blockchain node. It is a component of the blockchain node that
 exposes HTTP endpoints for external entities to send commands to the node. The
-proxy is notably used by the web clients to use the election system.
+proxy is notably used by the web clients to use the form system.
```
Comment (nkcr) [inline README.md:123]: here we should either simply say "use the system", or "use the voting system".

File: README.md [hunk 4]
```diff
@@ -120,12 +120,12 @@ The project has 4 main high-level components:
...
 
 **Web frontend** - The web frontend is a web app built with React. It offers a
 view for end-users to use the D-Voting system. The app is meant to be used by
 voters and admins. Admins can perform administrative tasks such as creating an
-election, closing it, or revealing the results. Depending on the task, the web
+form, closing it, or revealing the results. Depending on the task, the web
```
Comment (nkcr) [inline README.md:128]: "**an** election" -> "**a** form"
(same elsewhere)

File: README.md [hunk 5]
```diff
@@ -137,7 +137,7 @@ web-backend has a local database to store configuration data such as
 authorizations. Admins use the web-frontend to perform updates.
 
 **Blockchain node** - A blockchain node is the wide definition of the program
-that runs on a host and participate in the election logic. The blockchain node
+that runs on a host and participate in the form logic. The blockchain node
```
Comment (nkcr) [inline README.md:140]: "form logic" -> "voting logic".

File: README.md [hunk 6]
```diff
@@ -378,10 +378,10 @@ added to you path (like with `export PATH=$PATH:/Users/david/go/bin`).
 # Setup a simple system with 3 nodes
 
 If you are using Windows and cannot use tmux, you need to do the actions of the
-scripts in point *1* and *2* manually: open 3 terminal sessions and run the
-commands from the section *Run the nodes* below (1 command LLVL=info memcoin etc.
+scripts in point _1_ and _2_ manually: open 3 terminal sessions and run the
```
Comment (nkcr) [inline README.md:381]: we prefer using "*" instead of "_"

File: README.md [hunk 7]
```diff
@@ -528,18 +529,19 @@ LLVL=info memcoin --config /tmp/node1 e-voting scenarioTest --secretkey $sk
 
 You can also run scenario_test.go, by running in the integration folder this
 command:
+
```
Comment (nkcr) [inline README.md:532]: thanks 👍

File: contracts/evoting/evoting.go [hunk 8]
```diff
@@ -32,7 +32,7 @@ import (
 const (
 	shufflingProtocolName = "PairShuffle"
 	errGetTransaction     = "failed to get transaction: %v"
-	errGetElection        = "failed to get election: %v"
+	errGetForm        = "failed to get form: %v"
```
Comment (nkcr) [inline contracts/evoting/evoting.go:35]: it seems that your IDE is not formatting your go code for you. Please make sure it does by installing the recommended tooling for go code.

File: docs/ballot_encoding.md [hunk 9]
```diff
@@ -41,25 +41,25 @@ A possible encoding of an answer would be (by string concatenation):
...
 
-## Size of the ballot 
+## Size of the ballot
 
-In order to maintain complete voter anonymity and untraceability of ballots throughout the 
-election process, it is important that all encrypted ballots have the same size. To this aim, 
-the election has an attribute called "BallotSize" which is the size 
-that all ballots should have before they're encrypted. Smaller ballots should therefore be 
+In order to maintain complete voter anonymity and untraceability of ballots throughout the
+form process, it is important that all encrypted ballots have the same size. To this aim,
```
Comment (nkcr) [inline docs/ballot_encoding.md:52]: "voting process"

File: docs/introduction.md [hunk 10]
```diff
@@ -16,15 +16,15 @@ end-user can interact with the system.
 
 <p style="text-align:center"><img src="assets/layers.png"/></p>
 
-## Election flow
+## Form flow
```
Comment (nkcr) [inline docs/introduction.md:19]: "voting flow"

File: docs/introduction.md [hunk 11]
```diff
@@ -16,15 +16,15 @@ end-user can interact with the system.
...
 
-The following sequence diagram shows the entire flow of an election.
+The following sequence diagram shows the entire flow of an form.
 
-<p style="text-align:center"><img src="assets/election flow.png"/></p>
+<p style="text-align:center"><img src="assets/form flow.png"/></p>
```
Comment (nkcr) [inline docs/introduction.md:23]: You can generate the new sequence diagram but copy/pasting the source bellow there: https://sequencediagram.org/.

File: services/dkg/pedersen/mod.go [hunk 12]
```diff
@@ -166,9 +166,9 @@ type Actor struct {
 	factory     serde.Factory
 	service     ordering.Service
 	context     serde.Context
-	electionFac serde.Factory
+	formFac serde.Factory
```
Comment (nkcr) [inline services/dkg/pedersen/mod.go:169]: same comment about formatting. Your IDE should automatically do `gofmt` for you, which is the standard code formater for go.

**Review by nkcr — CHANGES_REQUESTED** [review]
That's a big chunk, thanks for the update! 

I made some comments, mostly about the documentation. Some parts need a bit of rephrasing, otherwise it can be weird.

Also, since this is updating pretty much everything in the code base, you should ensure that nothing got broken along the way. The CI already provides some guarantees but this is not enough. Did you try running the system manually with you changes ? You should ensure that all functionalities are still working, and the web frontend should work both with `nomock=on` or not (i.e. using the mocks).

File: docs/introduction.md (hunk 11 above)
Comment (Ghita2002) [reply docs/introduction.md:23]: Sorry, I'm not sure to understand. Did you mean that I need to past the link in the source line 23 ?

**Review by nkcr — APPROVED** [review]
Great work 👍

**Pull request merged**